offchain-modules/bridge-ckb-sdk/Cargo.lock

```diff
@@ -0,0 +1,2765 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Usually the cargo.lock is not tracked in rust libraries.
https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html